spanner/test/spannerTest.php

```diff
@@ -50,6 +50,12 @@ class spannerTest extends TestCase
     /** @var string lowCostInstanceId */
     protected static $lowCostInstanceId;
 
+    /** @var string instancePartitionInstanceId */
+    protected static $instancePartitionInstanceId;
+
+    /** @var Instance instancePartitionInstance */
+    protected static $instancePartitionInstance;
```

Comment:
nit, but you can typehint these directly now
```suggestion
    protected static Instance $instancePartitionInstance;
```